pkgs/development/interpreters/ruby/patchsets.nix

```diff
@@ -1,15 +1,21 @@
 { patchSet, useRailsExpress, ops, patchLevel, fetchpatch }:
 
 {
-  "2.7.5" = ops useRailsExpress [
-    "${patchSet}/patches/ruby/2.7/head/railsexpress/01-fix-broken-tests-caused-by-ad.patch"
-    "${patchSet}/patches/ruby/2.7/head/railsexpress/02-improve-gc-stats.patch"
-    "${patchSet}/patches/ruby/2.7/head/railsexpress/03-more-detailed-stacktrace.patch"
-  ];
-  "3.0.3" = ops useRailsExpress [
-    "${patchSet}/patches/ruby/3.0/head/railsexpress/01-improve-gc-stats.patch"
-    "${patchSet}/patches/ruby/3.0/head/railsexpress/02-malloc-trim.patch"
-  ];
-  "3.1.0" = ops useRailsExpress [ # no patches yet (2021-12-25)
-  ];
+  "2.7.5" =
+    [ ./skip-installing-bundled-gems-2.7.patch ]
+    ++ ops useRailsExpress [
+      "${patchSet}/patches/ruby/2.7/head/railsexpress/01-fix-broken-tests-caused-by-ad.patch"
+      "${patchSet}/patches/ruby/2.7/head/railsexpress/02-improve-gc-stats.patch"
+      "${patchSet}/patches/ruby/2.7/head/railsexpress/03-more-detailed-stacktrace.patch"
+    ];
+  "3.0.3" =
+    [ ./skip-installing-bundled-gems-2.7.patch ]
+    ++ ops useRailsExpress [
+      "${patchSet}/patches/ruby/3.0/head/railsexpress/01-improve-gc-stats.patch"
+      "${patchSet}/patches/ruby/3.0/head/railsexpress/02-malloc-trim.patch"
+    ];
+  "3.1.0" =
+    [ ./skip-installing-bundled-gems-3.1.patch ]
```

Comment:
Shouldn't we ship those bundled gems? I mean, we generally should provide an experience as closer as upstream wants, unless we have a better justification.

But maybe this discussion should be made in another issue/PR. I am fine in keeping this here for now.